Forbid adding a sub-column to an object(ignored) if table isn't empty (#16908)
crate/crate: docs/appendices/release-notes/5.9.3.rst

      |       1.0 |
      +-----------+

> - Fixed an issue which can cause that the data cannot be queried anymore and the
>   :ref:`analyze` to fail when adding a sub column to an object column with
>   column policy ``IGNORED`` when the table is not empty and the new column's
>   data type is different from the existing column's values.

Reviewer: Maybe it reads a bit better like:
```suggestion
- Fixed an issue which can cause the data of a table to not be queried anymore and the
  :ref:`analyze` to fail, if a sub column is added to an object column with
  column policy ``IGNORED`` when the table is not empty and the new column's
  data type is different from the existing column's values.
```
?

  This is not allowed anymore and results in an error.

>   For example::

Reviewer: super minor, this can be moved to the prev line.
